ganttproject/src/main/java/biz/ganttproject/storage/cloud/GPCloudHttpImpl.kt

```diff
@@ -290,6 +304,7 @@ class WebSocketClient {
       when (it) {
         "ProjectLockStatusChange" -> fireLockStatusChange(payload)
         "ProjectChange", "ProjectRevert" -> fireContentsChange(payload)
+        "XlogReceived" -> fireXlogReceived(payload)
```

Comment:
Just "Xlog"? It is obviously received here...

Reply:
Sorry, I don't get it
The server sends back _something_ which should contain the id of our last log it has saved. Do you hint that the name is bad? :) And why "obviously received"?..